src/json_validate.py

```diff
@@ -112,15 +112,16 @@ def validate_trial_generate_json(trial_generate_json):
             validate(instance=trial_generate_json, schema=delete_experiment_schema, format_checker=draft7_format_checker)
         elif not str(trial_generate_json["operation"]) or not str(trial_generate_json["operation"]).strip() or \
                 trial_generate_json["operation"] is None:
-            errorMsg = "Parameters" + HPOErrorConstants.VALUE_MISSING
+            errorMsg = "Parameters" + "operation" + HPOErrorConstants.VALUE_MISSING
         else:
             errorMsg = HPOErrorConstants.INVALID_OPERATION
 
         return errorMsg
     except jsonschema.exceptions.ValidationError as err:
         # Check if the exception is due to empty or null required parameters and prepare the response accordingly
         if any(word in err.message for word in HPOErrorConstants.JSON_NULL_VALUES):
-            errorMsg = "Parameters" + HPOErrorConstants.VALUE_MISSING
+            words = ' '.join([word for word in HPOErrorConstants.JSON_NULL_VALUES if word in err.message])
+            errorMsg = "Parameters" + words + HPOErrorConstants.VALUE_MISSING
```

Comment:
@shreyabiradar07 thanks for catching that, now I've defined a variable outside of the `if` statement so should be able to use it in the error message. Can you help me figure out how to test this  or test it for me?

I'm not yet familar with how to build/run in development mode -- just been using the `./deploy_hpo.sh -c docker`

FWIW I finally figured out that my problem was that `experiment_id` needs to be a string, not an integer. That was my problem but the error message was saying that I had some null or missing parameters, and was not really enough for me to figure out what was going on.

Reply:
@kdelee - thanks for pointing, the error should ideally be type mismatch instead of empty or null value when experiment_id  is an integer . On reproducing the latest changes with search space "experiment_id":5  it throws the below error:
Parameters is not of type 'string' cannot be empty or null!
Ideally the error should be - Parameter 'experiment_id' - 5 is not of type 'string'

I'm testing the changes using REST client POSTMAN with rest service url http://localhost:8085/ exposed by hpo using ./deploy_hpo.sh script.

Please checkoutbuild_hpo script to build kruize hpo with a default/custom container image, using the command ./build_hpo.sh -d -o <custom-container-image> wherein -d flag is to specify dev-friendly mode, -o flag to specify custom docker/quay image(by default it picks up kruize hpo image) ./build_hpo.sh --help for more details on the optional flags to be used

Reply:
@kdelee - I have created a [PR](https://github.com/kruize/hpo/pull/138) to fix the parameter data type mismatch validation error. Let me know if this fixes your problem, if not please provide me with details to reproduce the problem faced.